props/nested-components.md

```diff
@@ -0,0 +1,195 @@
+# Nested Components
+
+## Goals
+
+In order to increase the complexity of our React webapps, we should practice nesting components.
+
+Nested components share a relationship that is different from sibling components.
+
```

Comment:
I don't see anywhere that a sense of the relationship between siblings was ever established and there's no discussion of differences here, so I would cut this.
```suggestion
```